doc/user/client.md

```diff
@@ -0,0 +1,252 @@
+# Operator-SDK: Controller Runtime Client API
+
+## Overview
+
+The [kubernetes-sigs][org-kubernetes-sigs] organization builds tools to call the [Kubernetes][site-kubernetes] API in a clean, abstracted manner, much like the Operator SDK. The [controller-runtime][repo-controller-runtime] project is meant to help users build [controllers][site-kubernetes-controllers] easily by generating a controller project that interacts with a k8s cluster via CRUD operations. User-defined controller can perform specific tasks in a cluster; [operators][site-operators] can use multiple controllers to perform a variety of tasks. As operators use at least one controller, the SDK can rely on controller-runtime's k8s API code rather than develop a parallel set of API calls to execute the same cluster operations, namely CRUD operations.
+
+controller-runtime defines several interfaces used for cluster interaction:
+- `client.Client`: implementers perform CRUD operations on a k8s cluster.
+- `manager.Manager`: manages shared dependencies, such as Caches and Clients.
+- `reconcile.Reconciler`: compares provided state with actual cluster state and updates the cluster on finding state differences using a Client.
+
+## Client Usage
+
+### Default Client
+
+The SDK relies on a `manager.Manager` to create a `client.Client` interface that performs Create, Update, Delete, Get, and List operations within a `reconcile.Reconciler`'s Reconcile function. The SDK will generate code to create a Manager, which holds a Cache and a Client to be used in CRUD operations and communicate with the API server. By default a Controller's Reconciler will be populated with the Manager's Client which is a [split-client][doc-split-client].
+
+`pkg/controller/<kind>/<kind>_controller.go`:
+```Go
+func newReconciler(mgr manager.Manager) reconcile.Reconciler {
+	return &ReconcileKind{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+}
+
+type ReconcileKind struct {
+	// Populated above from a manager.Manager.
+	client client.Client
+	scheme *runtime.Scheme
+}
+```
+
+A split client reads (Get and List) from the Cache and writes (Create, Update, Delete) to the API server. Reading from the Cache significantly reduces request load on the API server; as long as the Cache is updated by the API server, read operations are eventually consistent. 
+
+### Non-default Client
+
+An operator developer may wish to create their own Client with different read/write optimzations, for example. controller-runtime provides a [constructor][doc-client-constr] for Clients:
```

Comment:
We can be more specific here:
`with different read/write optimzations`==>`that serves read requests(Get List) from the apiserver instead of the cache`